core/src/main/java/org/fao/geonet/kernel/setting/Settings.java

```diff
@@ -45,7 +45,7 @@
     public static final String SYSTEM_PROXY_USERNAME = "system/proxy/username";
     public static final String SYSTEM_PROXY_PASSWORD = "system/proxy/password";
     public static final String SYSTEM_PROXY_IGNOREHOSTLIST = "system/proxy/ignorehostlist";
-    public static final String SYSTEM_LUCENE_IGNORECHARS = "system/requestedLanguage/ignorechars";
+    public static final GNSetting SYSTEM_LUCENE_IGNORECHARS = new GNSetting("system/requestedLanguage/ignorechars", true);
```

Comment:
I just fixed the settings that were adding lots of useless lines in the log files. Changing all the settings is out of the scope of this issue.